.readthedocs.yaml

```diff
@@ -22,6 +22,6 @@ python:
   version: 3.8
   install:
     - requirements: docs/readthedocs-requirements.txt
-    - requirements: docs-requirements.txt
     - method: pip
       path: .
+    - requirements: docs-requirements.txt
```

Comment:
This is what did it

Reply:
It should be ok to remove the MarkupSafe requirement from `docs-requirements` as well, since `nbsphinx>=0.8.5`, which is present in the docs requirements, also addresses the Markupsafe issue in its own requirements

Reply:
Ah thanks, will do